contrib/bump-version.sh

```diff
@@ -4,24 +4,35 @@
 # the CURRENT_VERSION to the TARGET_VERSION. This is meant as a helper - but
 # probably should still double-check the changes are correct
 
-if [ $# -ne 1 ]; then
+if [ $# -ne 1 ] || [ `expr match $1 ".*_.*"` == 0 ]; then
     echo "USAGE: $0 <target-version>"
-    echo "  example: $0 'v1.3.4_coreos.0'"
+    echo "  example: $0 'v1.3.5_coreos.1'"
     exit 1
 fi
 
-CURRENT_VERSION=${CURRENT_VERSION:-"v1.3.4_coreos.0"}
+CURRENT_VERSION=${CURRENT_VERSION:-"v1.3.5_coreos.1"}
 TARGET_VERSION=${1}
 
+CURRENT_INDEX=`expr index $CURRENT_VERSION _`
+TARGET_INDEX=`expr index $TARGET_VERSION _`
+
+CURRENT_VERSION_BASE=${CURRENT_VERSION:0:$CURRENT_INDEX -1}
+TARGET_VERSION_BASE=${TARGET_VERSION:0:$TARGET_INDEX -1}
+
+CURRENT_VERSION_BUILD=${CURRENT_VERSION:$CURRENT_INDEX}
+TARGET_VERSION_BUILD=${TARGET_VERSION:$TARGET_INDEX}
```

Comment:
I appreciate the effort here, but at this point I think lets just leave it as is and maybe an echo to the end along the lines of:

NOTE: grep for major.minor version, this script may not catch all changes: git grep "v1\.3\."

I say this because there are examples of extraneous version changes in this PR alone. The script wasn't meant to be perfect, just reduce the annoyance.